API_v1.md

```diff
@@ -689,15 +927,15 @@ Create a new user with a place and a contact.
 
 ### Permissions
 
-`can_create_users`  
+`can_create_users`, `cat_create_places`, `cat_create_contacts`
```

Comment:
Probably by default anyone can create places and people, right? That's the way the webapp works currently, and there's no reason why they should be able to do it there and not through the API.